### DISCUSSION TITLE
Minimal code for reading pointclouds in bag file and visualizing them with Open3D

### DISCUSSION THREAD

**Discussion by evbernardes** [isl-org/Open3D#6352] [Q&A]
I'm starting to work with a Realsense camera and I'd like to use Open3D to do some processing of pointclouds.

I didn't manage to find any examples that do the following:

1. Open bag file with recorded video from Realsense
2. Transform depth_images into pointclouds
3. (Optional) do something with pointclouds
4. Show pointclouds on a visualization window
5. Repeat for next frame until the end

Can someone share a minimal code that does this?

**Comment by ssheorey** [comment]
Hi @evbernardes here's an example for what you are looking for:

https://github.com/isl-org/Open3D/blob/master/examples/python/io/realsense_io.py

A more complex example is here:

https://github.com/isl-org/Open3D/blob/master/examples/python/visualization/online_processing.py
